client/src/components/Ratings.jsx

```diff
@@ -44,21 +56,53 @@ function Ratings ({product_Id, productName}) {
     };
     fetchMeta();
   }, []);
+
+  useEffect(() => {
+    let filtering = [];
+    for (const star in starsClicked) {
+      if (starsClicked[star] === true) {
+       filtering.push(parseInt(star));
+      }
+    }
+    setStarsFilter(filtering);
+  }, [starsClicked]);
+
+  useEffect(() => {
+    let newList;
+    if (starsFilter.length > 0) {
+      newList = productReviews.results.filter(result =>
+        starsFilter.includes(result.rating)
+      );
+      setReviews(newList);
+    } else {
+      setRevert(!revert);
+    }
+  },[starsFilter]);
+
+  const handleChangeSort = (sortedBy) => {
+    setSort(sortedBy);
+  };
+
```

Comment:
Unsure what is happening but when I sort by newest, the newer messages do show up but are weirdly centered instead of Left-aligned. Something to do with my pulldown of your stuff or is this really what it looks like? Hope you can see the image below.

![image](https://user-images.githubusercontent.com/85661360/147868908-33710822-1b8c-498b-be54-26fa6ba1342c.png)

Reply:
its justifying based on the size of the body. Someone from another team is sending reviews to the server that is not abiding the 50 character limit. that limit makes all messages a uniform size. it can be solved later when we touch up the css in everyones section.